reqs/3.6/requirements-dev.txt

```diff
@@ -160,6 +161,7 @@ sphobjinv==2.0.1
 sqlalchemy==1.3.22
 stdlib-list==0.8.0
 stevedore==3.3.0
+sympy==1.7.1
```

Comment:
Dp we really want to make the adr add new requirements?

Reply:
Yes, indeed a bit strange. Currently I only need it for the sympy notebook. But actually the ADR is way too elaborate already.

Reply:
I would say its best if the adrs do not modify the requirements

Reply:
See 693d1fd and 1a5dbcc